webapp/src/selectors/index.ts

```diff
@@ -13,17 +13,19 @@ const getPluginState = (state) => state['plugins-' + PluginId] || {};
 
 export const getPluginServerRoute = (state) => {
     const config = getConfig(state);
-
     let basePath = '';
+    let finalPath = '';
+
     if (config && config.SiteURL) {
         basePath = new URL(config.SiteURL).pathname;
-
         if (basePath && basePath[basePath.length - 1] === '/') {
             basePath = basePath.substr(0, basePath.length - 1);
         }
-    }
+        const origin = new URL(config.SiteURL).origin;
 
-    return basePath + '/plugins/' + PluginId;
+        finalPath = `${origin}${basePath}/plugins/${PluginId}`;
```

Comment:
@hanzei  not sure if this is the right style you are looking for but it works.

Reply:
yeah base path starts with a slash. Come to think of it, I should remove this commit from this PR. The idea was to have the full path everywhere. Does it make sense to keep it or should we skip this out from this pr ?